my_list.rb

```diff
@@ -0,0 +1,13 @@
+require './my_enumerable'
```

Comment:
- [ ] It is a convention to use the `require_relative` instead of the `require` for importing other project files.

> "Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory."